pkg/scheduler/plugins/elasticquota/controller.go

```diff
@@ -62,6 +62,22 @@ func (ctrl *Controller) Name() string {
 func (ctrl *Controller) Start() {
 	go wait.Until(ctrl.syncElasticQuotaStatusWorker, 1*time.Second, context.TODO().Done())
 	go wait.Until(ctrl.syncElasticQuotaStatusMetricsWorker, 10*time.Second, context.TODO().Done())
+	if ctrl.plugin.pluginArgs.EnableRuntimeQuota {
+		go wait.Until(ctrl.syncElasticQuotaRuntimeWorker, 10*time.Second, context.TODO().Done())
+	}
+}
+
+func (ctrl *Controller) syncElasticQuotaRuntimeWorker() {
+	elasticQuotas, err := ctrl.plugin.quotaLister.List(labels.Everything())
+	if err != nil {
+		klog.V(3).ErrorS(err, "Unable to list elastic quota in syncElasticQuotaRuntimeWorker")
+		return
+	}
+	for _, eq := range elasticQuotas {
+		if !extension.IsParentQuota(eq) {
+			ctrl.plugin.groupQuotaManager.RefreshRuntime(eq.Name)
```

Comment:
In the Multi-Root scenario, there may be multiple groupQuotaManagers, so you need to obtain groupQuotaManager on demand. You can refer to how syncElasticQuotaStatusWorker handles this.